dao/src/main/java/greencity/entity/table/TableColumnWidthForEmployee.java

```diff
@@ -11,80 +14,89 @@
 @AllArgsConstructor
 @NoArgsConstructor
 @Builder
-@EqualsAndHashCode
-@ToString
 public class TableColumnWidthForEmployee {
+    private static final Integer TABLE_DEFAULT_COLUMN_WIDTH = 120;
```

Comment:
I think the constant name could be changed to `DEFAULT_WIDTH`
We have already known from the class name that it is for `Table Column Width`